routes/api/collectors.js

```diff
@@ -0,0 +1,64 @@
+var express = require('express');
+var router = express.Router();
+var gcloud = require('gcloud');
+
+var datastore = gcloud.datastore({
+  projectId: 'prototype-content-consumer',
+  keyFilename: './credentials/private-key.json'
+});
+
+/* fields => {
+               productname: chair,
+               hostname: www.ikea.com,
+               pageurl: http://www.ikea.com/us/en/catalog/products/S89100143/,
+               ...
+             }
+*/
+
+function saveCollector(fields) {
+  let collectorKey = datastore.key('Collector');
+  // TODO: created at / updated at fields
+  datastore.save({
+    key: collectorKey,
+    data: fields
+  }, function (err) {
+    if (err) {
+      return res.json({errors: req});
+    }
+  });
+}
+
+/* POST /api/v1/collectors
+   body: {
+    hostname=www.ikea.com,
+    productname=sweater,
```

Comment:
How about we consider removing productname and pageurl, then replace them with a payload object.

e.g.
- host
- payload

We should also consider passing a "nonce" with this body. At first, we do not need to use the nonce, but at least have it in the signature.